src/Drivers/miniqmc.cpp

```diff
@@ -338,6 +350,9 @@ int main(int argc, char **argv)
     for (int mc = 0; mc < nsteps; ++mc)
     {
       Timers[Timer_Diffusion]->start();
+      Timers[Timer_Determinant]->start();
+      determinant->recompute();
```

Comment:
recompute happens inside wavefunction->evaluateLog. No need here.

Reply:
okay. Then should there be a call to wavefunction->evaluateLog inside the mc step loop (but outside substeps)?